### PR TITLE
Check and return variable session timeout info

### PR DESCRIPTION
**Pull request by 1000TurquoisePogs**
Updates ZSS for the feature in https://github.com/zowe/zowe-common-c/pull/194

### CODE REVIEW

File: c/zss.c [hunk 1]
```diff
@@ -413,9 +414,20 @@ static hashtable *getServerUserTimeoutsHt(ShortLivedHeap *slh, const char *filen
...
         char *userKey = jsonPropertyGetKey(property);
+        strupcase(userKey); /* make case insensitive */
         int timeoutValue = jsonObjectGetNumber(users, userKey);
+        if (!strcmp(key, "groups")) {
+          int gid = groupIdGet(userKey, &rc, &rsn);
+          if (rc == 0) {
+            char *gidString = safeMalloc(GID_MAX_CHAR_LENGTH+1, "groupstring");
+            convertIntToString(gidString, GID_MAX_CHAR_LENGTH, gid);
+            gidString[GID_MAX_CHAR_LENGTH]='\0';
+            htPut(ht, gidString, (void*)timeoutValue);
```
Comment (lchudinov) [inline c/zss.c:425]: `htPut(ht, gidString, POINTER_FROM_INT(timeoutValue));`

Comment (lchudinov) [reply c/zss.c:425]: Please use `POINTER_FROM_INT` and `INT_FROM_POINTER`.

**Review by DivergentEuropeans — COMMENTED** [review]
Functionally this now works good for me ✔️ 
 
 ZSS_LOG_PARS_ZSS_TIMEOUT_MSG message should say something like,
"Session timeouts for '%s' N/A, file not provided or incorrectly formatted JSON. ZSS sessions will use the default length of one hour.\n"
 instead of
"Session timeouts for '%s' N/A, file not provided. ZSS sessions will use the default length of one hour.\n"
since I get that message if I forget a comma or something in my timeouts file

**Review by lchudinov — APPROVED** [review]
Works fine to me.

**Pull request merged**